modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -208,96 +169,25 @@
-                         match_requests_on: %i[method uri]) do
-          VCR.use_cassette('mobile/providers/get_provider_200', match_requests_on: %i[method uri], tag: :force_utf8) do
-            allow_any_instance_of(Mobile::V2::Appointments::ProviderNames).to \
-              receive(:fetch_provider).and_return(provider_response)
-            get '/mobile/v0/appointments', headers: sis_headers, params:
-          end
-        end
-        expect(response).to have_http_status(:ok)
-        expect(JSON.parse(response.body)['data'].size).to eq(980)
-        # VAOS v2 appointment is only different from appointments by allowing some fields to be nil.
-        # This is due to bad staging data.
-        expect(response.body).to match_json_schema('VAOS_v2_appointments')
-
-        uniq_statuses = response.parsed_body['data'].map { |appt| appt.dig('attributes', 'status') }.uniq
-        expect(uniq_statuses).to match_array(%w[CANCELLED BOOKED SUBMITTED])
-
-        proposed_times = response.parsed_body['data'].map { |appt| appt.dig('attributes', 'proposedTimes') }.uniq
-        expect(proposed_times).not_to eq([nil])
-
-        is_pending = response.parsed_body['data'].map { |appt| appt.dig('attributes', 'isPending') }.uniq
-        expect(is_pending).to match_array([true, false])
-      end
-    end
-
     context 'request telehealth onsite appointment' do
-      before do
-        mock_facility
-        mock_clinic
-      end
-
       let(:start_date) { Time.zone.parse('1991-01-01T00:00:00Z').iso8601 }
       let(:end_date) { Time.zone.parse('2023-01-01T00:00:00Z').iso8601 }
       let(:params) do
         { page: { number: 1, size: 9999 }, startDate: start_date, endDate: end_date }
       end
 
       it 'processes appointments without error' do
-        VCR.use_cassette('mobile/appointments/VAOS_v2/get_appointment_200_telehealth_onsite',
-                         match_requests_on: %i[method uri]) do
-          VCR.use_cassette('mobile/providers/get_provider_200', match_requests_on: %i[method uri], tag: :force_utf8) do
-            allow_any_instance_of(Mobile::V2::Appointments::ProviderNames).to \
-              receive(:fetch_provider).and_return(provider_response)
-            get '/mobile/v0/appointments', headers: sis_headers, params:
+        VCR.use_cassette('mobile/appointments/VAOS_v2/get_clinic_200', match_requests_on: %i[method uri]) do
+          VCR.use_cassette('mobile/appointments/VAOS_v2/get_facility_200', match_requests_on: %i[method uri]) do
+            VCR.use_cassette('mobile/appointments/VAOS_v2/get_appointment_200_telehealth_onsite',
+                             match_requests_on: %i[method uri]) do
+              VCR.use_cassette('mobile/providers/get_provider_200', match_requests_on: %i[method uri],
+                                                                    tag: :force_utf8) do
+                allow_any_instance_of(Mobile::V2::Appointments::ProviderNames).to \
```

Comment:
Why are we using the `get_provider_200` cassette above and also stubbing the provider names method here? Shouldn't only one of them be necessary? If it's possible to do this without stubbing, that is preferred.

Reply:
Thanks for pointing that out. I removed the stubbing.